frontend/src/features/rollout-announcement/components/NewFeatureContent.tsx

```diff
@@ -41,13 +48,16 @@ export const NewFeatureContent = (props: {
   content: NewFeatureContentProps
 }): JSX.Element => {
   const { title, description, animationData } = props.content
+  const isMobile = useIsMobile()
 
   return (
     <>
       <LottieAnimation
         bg="primary.100"
         pt="4.5rem"
+        height={isMobile ? '33vh' : ''}
```

Comment:
nit: prefer using chakra's responsive styles syntax whenever possible (using isMobiles requires some computation of current width instead of using media queries)

```tsx
height={['33%']}
// not sure it the above will work properly, if not can try
// height={{ base: 33%, md: 'initial' }}
```